src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/linux/LinuxAArch64Linker.java

```diff
@@ -41,13 +41,17 @@
  * the ARM 64-bit Architecture".
  */
 public final class LinuxAArch64Linker extends AbstractLinker {
-    private static LinuxAArch64Linker instance;
 
     public static LinuxAArch64Linker getInstance() {
```

Comment:
I think the `getInstance()` method should only return `Linker`.

Reply:
It could. We typically try to be as precise as possible with internal methods though.